packages/calling-stateful-client/src/CallContext.ts

```diff
@@ -275,19 +275,13 @@ export class CallContext {
   }
 
   /* @conditional-compile-remove(video-background-effects) */
-  public setCallLocalVideoStreamVideoEffects(
-    callId: string,
-    videoEffects: Partial<LocalVideoStreamVideoEffectsState>
-  ): void {
+  public setCallLocalVideoStreamVideoEffects(callId: string, videoEffects: LocalVideoStreamVideoEffectsState): void {
```

Comment:
should the videoEffects be of type `videoeffectname[]` not sure if we still need `LocalVideoStreamVideoEffectsState`

Reply:
Great point, I also thought upon this. My feeling for keeping the nested `videoEffects: LocalVideoStreamVideoEffectsState` in the stateful client was that the videoEffects returned by the calling sdk is an object. Currently it just has activeEffects array, but in future there is a potential to have more properties